scripts/bug_classifier.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+import argparse
+import os
+
+import numpy as np
+
+from bugbug import bugzilla
+from bugbug.models import get_model_class
+
+
+def classify_bugs(model_name):
+    model_file_name = f"{model_name}model"
```

Comment:
In the previous code, we were sometimes adding the classifier to the name, we should do the same here.

Reply:
The classifier type only applies to the component_nn right? Can't the user just pass in component_nn as the model, instead of component --classifier nn?

Reply:
> The classifier type only applies to the component_nn right?

For now, yes.

> Can't the user just pass in component_nn as the model, instead of component --classifier nn?

They could, but it's less clear for them and there's no advantage for us in doing that.